docs/sql-ref-syntax-qry-sampling.md

```diff
@@ -19,4 +19,74 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+
+The `TABLESAMPLE` statement is used to sample the table. It supports the following sampling methods:
+- `TABLESAMPLE`(x `ROWS`): Sample the table down to the given number of rows.
+- `TABLESAMPLE`(x `PERCENT`): Sample the table down to the given percentage. Note that percentages are defined as a number between 0 and 100.
+- `TABLESAMPLE`(`BUCKET` x `OUT OF` y): Sample the table down to a `x` out of `y` fraction.
+
+Note: `TABLESAMPLE` returns the approximate number of rows or fraction requested.
+
+
+### Syntax
+{% highlight sql %}
+    TABLESAMPLE ((integer_expression | decimal_expression) PERCENT)
+    | TABLESAMPLE (integer_expression ROWS)
+    | TABLESAMPLE (BUCKET integer_expression OUT OF integer_expression)
+{% endhighlight %}
+
+### Examples
+{% highlight sql %}
+SELECT * FROM test;
++--+----+
+|id|name|
++--+----+
+|5 |Alex|
```

Comment:
btw, why does the result is left-aligned? The output of `showString` seems to be right-aligned?
```
scala> sql("SELECT * FROM test;").show()
+---+----+
| id|name|
+---+----+
|  1| aaa|
+---+----+
```